test/files/qty000/generate.py

```diff
@@ -0,0 +1,24 @@
+import ifcopenshell
+import ifcopenshell.template
+import os
+
+rule_code = "qty000"
+
+abs_path = os.path.join(os.getcwd(), "test", "files", rule_code)
+save_ifc_file = lambda file, filename: file.write(os.path.join(abs_path, filename))
+
+f = ifcopenshell.template.create(schema_identifier="IFC4X3_ADD2")
+
+save_ifc_file(f, 'pass-qty000-not_activated_no_quantity.ifc')
+
+qt = f.createIfcElementQuantity()
```

Comment:
I would also create a model with a normal property set just to double check that it is indeed quantities that trigger this. Just because they use the same association mechanism of ifcreldefines.

Reply:
I've [added two more files](https://github.com/buildingSMART/ifc-gherkin-rules/pull/247/commits/db0a2bcbff354ef5ea3e46deb9d3fc98ab4fee03) from a file that was passing the schema validation. [The first of those two file](https://github.com/buildingSMART/ifc-gherkin-rules/pull/247/commits/db0a2bcbff354ef5ea3e46deb9d3fc98ab4fee03#diff-e11c79566434708bb375016469b8f06bdecb72b599ce21ac5f188de12d355eed)s also includes psets, however only the relating property definition with type IfcElementQuantity activate the rule.
```
context.gherkin_outcomes
[<ValidationOutcome: ...=EXECUTED>, <ValidationOutcome: ...ty=PASSED>]
get_stack_tree(context)[1]
[(), (), (), (), (#40=IfcPropertySet('...,#38,#39)), #42=IfcElementQuanti...$,$,(#41))), (#70=IfcPropertySet('...,#68,#69)), #78=IfcPropertySet('...,#76,#77)), #81=IfcPropertySet('...(#79,#80)), None)]
```

When I remove this quantity in [another testfile ](https://github.com/buildingSMART/ifc-gherkin-rules/pull/247/commits/db0a2bcbff354ef5ea3e46deb9d3fc98ab4fee03#diff-edffafe18b7201b844b6453967dc2dfa534815078f46a7d112156a92f3aebe2b)the rule doesn't get activated, hence the entity stack. The file has 4 instances of IfcObject. Only the last two (IfcPlate, IfcAssembly) have related properties. However, none of them is an element quantity.
![image](https://github.com/user-attachments/assets/5552f398-f606-41d8-bef5-fe32bdc4de37)